src/openstudio_lib/GeometryPreviewView.cpp

```diff
@@ -102,27 +100,74 @@ PreviewWebView::PreviewWebView(bool isIP, const model::Model& model, QWidget* t_
   m_refreshBtn->setVisible(true);
 
   m_view = new QWebEngineView(this);
-  m_view->settings()->setAttribute(QWebEngineSettings::WebAttribute::LocalContentCanAccessRemoteUrls, true);
-  m_view->settings()->setAttribute(QWebEngineSettings::WebAttribute::SpatialNavigationEnabled, true);
-
   m_page = new OSWebEnginePage(this);
   m_view->setPage(m_page);  // note, view does not take ownership of page
 
+
   connect(m_view, &QWebEngineView::loadFinished, this, &PreviewWebView::onLoadFinished);
-  //connect(m_view, &QWebEngineView::loadProgress, this, &PreviewWebView::onLoadProgress);
```

Comment:
These drive the progress bar when loading right?  That is useful for feedback when loading large models, can we keep these lines?